sdk/core/Azure.Core/generate_solution.ps1

```diff
@@ -1,21 +1,39 @@
+Set-StrictMode -Version Latest
+
 $RepoRoot = Resolve-Path $PSScriptRoot\..\..\..
 $ObjDirectory = "$RepoRoot\artifacts\obj";
-dotnet restore $RepoRoot\eng\service.proj
+#dotnet restore $RepoRoot\eng\service.proj
```

Comment:
Was this line intentionally commented out?

Reply:
Nope, that was part of testing.